### ISSUE TITLE
Dependency 'handlebars' missing

### ISSUE THREAD

**Issue by PrimaryFeather** [soimy/msdf-bmfont-xml#4]
After installing the tool, any attempt to run it produced the following error:

```
module.js:491
    throw err;
    ^

Error: Cannot find module 'handlebars'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/msdf-bmfont-xml/cli.js:7:20)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
```

It was fixed by installing _handlebars_ manually via `npm install handlebars`. But I guess you can include that in the list of dependencies, right?

**Comment by soimy** [comment]
Thank you for this bug report! Forget to move `handlebars` from `devDependencies` to `dependencie`
This is fixed in commit 369e582c8884274d807d822cd1d0276e0bd5ff38

**Comment by PrimaryFeather** [comment]
Perfect! Thanks a lot for the quick fix! :smile:
